netci.groovy

```diff
@@ -1693,25 +1720,24 @@ def static calculateBuildCommands(def newJob, def scenario, def branch, def isPR
 
                         def machineAffinityOptions = ['use_arm64_build_machine' : true]
                         setMachineAffinity(newJob, os, architecture, machineAffinityOptions)
-                        // For checked runs we will also run testing.
+
                         buildCommands += "set __TestIntermediateDir=int&&build.cmd ${lowerConfiguration} ${buildArchitecture} -priority=1"
+
+                        // Also run testing.
                         buildCommands += "python tests\\scripts\\arm64_post_build.py -repo_root %WORKSPACE% -arch ${buildArchitecture} -build_type ${lowerConfiguration} -scenario ${scenario} -testarch ${architecture} -key_location C:\\tools\\key.txt"
                     }
-                    else if (lowerConfiguration == "release") {
-                        buildCommands += "set __TestIntermediateDir=int&&build.cmd ${lowerConfiguration} ${buildArchitecture}"
-                    }
                     // Add archival.
                     Utilities.addArchival(newJob, "bin/Product/**", "bin/Product/**/.nuget/**")
                     break
                 case 'arm64':
-                    assert (scenario == 'default') || (scenario == 'pri1r2r') || (scenario == 'gcstress0x3') || (scenario == 'gcstress0xc')
+                    assert Constants.validArmWindowsScenarios.contains(scenario)
                    
                     // Set time out
                     setTestJobTimeOut(newJob, scenario)
 
                     // Debug runs take too long to run. So build job only.
                     if (lowerConfiguration == "debug") {
-                       buildCommands += "set __TestIntermediateDir=int&&build.cmd ${lowerConfiguration} ${architecture} toolset_dir C:\\ats2"
+                       buildCommands += "set __TestIntermediateDir=int&&build.cmd ${lowerConfiguration} ${architecture} toolset_dir C:\\ats2 -priority=1"
```

Comment:
This seems unnecessary. Wont we be adding a bunch of time to the jump building tests we will never run?

Reply:
I can revert this if you like. It just seemed weird to me that we build the pri 1 tests for checked, and for debug for arm, but not here. Why not be consistent?

Reply:
I think I would prefer to keep this as it is, I understand the consistency argument. However, we do not use these tests and the time added to the build is significant.